index.js

```diff
@@ -44,14 +44,41 @@ exports.run = function(runner, specs) {
     });
   });
 
+  function guid() {
+    function s4() {
+      return Math.floor((1 + Math.random()) * 0x10000)
+        .toString(16)
+        .substring(1);
+    }
+    return s4() + s4() + '-' + s4() + '-' + s4() + '-' + s4() + '-' + s4() + s4() + s4();
+  }
+
   function convertOptionsToCliArguments(options) {
     var cliArguments = ['node', 'cucumberjs'];
+    var config = runner.getConfig();
 
     for (var option in options) {
       var cliArgumentValues = convertOptionValueToCliValues(option, options[option]);
 
       if (Array.isArray(cliArgumentValues)) {
         cliArgumentValues.forEach(function (value) {
+          if (config.capabilities.shardTestFiles) {
+            if (option == 'format') {
+              var parts = value.split(':');
```

Comment:
It's evaluated in your config, but by the time it gets down to this library it's just a string. Right?